### PR TITLE
Explorer UI 

### PR DESCRIPTION
**Pull request by timpau**
- brand page
- shapes and gradients
- custom fonts

---
This change is [Reviewable](https://reviewable.io/reviews/certusone/wormhole/820)

### CODE REVIEW

File: explorer/src/components/Navbar.tsx [hunk 1]
```diff
@@ -9,6 +10,30 @@ const linkStyle = { ml: 3, textUnderlineOffset: 6 };
...
+      flexWrap: 'wrap',
+      justifyContent: 'center',
+      alignItems: 'center',
+      background: '#17153f',
+      textAlign: 'center',
+      p: 1
+    }}>
+      <Box sx={{ m: '5px 10px' }}>ImmuneFi bug bounty</Box>
+      <Button
+          component={RouterLink}
```
Comment (evan-gray) [inline explorer/src/components/Navbar.tsx:25]: This should be an `OutboundLink` if it goes to an external resource

File: explorer/src/components/Navbar.tsx [hunk 2]
```diff
@@ -9,6 +10,30 @@ const linkStyle = { ml: 3, textUnderlineOffset: 6 };
...
+          to="https://www.immunefi.com/bounty/wormhole/"
```
Comment (evan-gray) [inline explorer/src/components/Navbar.tsx:26]: This should be `href` if it goes to an external resource

File: explorer/src/components/SEO.tsx [hunk 3]
```diff
@@ -1,6 +1,7 @@
 import React from 'react'
 import { Helmet, HelmetProps } from 'react-helmet'
 import { useStaticQuery, graphql } from 'gatsby'
+import Suisse from '../fonts/SuisseBPIntlBold.woff2'
```
Comment (evan-gray) [inline explorer/src/components/SEO.tsx:4]: Fonts should be added in `plugins/gatsby-plugin-top-layout/TopLayout.js`, not here.

File: explorer/src/components/Footer.tsx [hunk 4]
```diff
@@ -136,6 +155,14 @@ const Footer = () => (
               >
                 Blog
               </Link>
+              <Link
+                href={brand}
```
Comment (evan-gray) [inline explorer/src/components/Footer.tsx:159]: This is an internal link so should use `component={RouterLink}` and `to` like the above.

File: explorer/src/components/GridWithCards.tsx [hunk 5]
```diff
@@ -97,17 +102,18 @@ const GridWithCards = ({
...
+                width: size,
+                height: size,
               }}
             >
               <img src={src} alt="" style={imgStyle} />
             </Box>
-            <Typography variant="h4" textAlign={headerTextAlign}>
-              {header}
-            </Typography>
+            <Typography variant="h4" textAlign={headerTextAlign} dangerouslySetInnerHTML={{__html: header}} />
```
Comment (evan-gray) [inline explorer/src/components/GridWithCards.tsx:116]: please do not use `dangerouslySetInnerHTML`

**Review by evan-gray — CHANGES_REQUESTED** [review]

**Comment by justinschuldt** [comment]
I took the explorer changes from this branch and applied them to a fresh copy of the repo. I'm not sure where all the extra commits came from, or how to make this branch clean. @timpau please check out #902.

**Comment by timpau** [comment]
> I took the explorer changes from this branch and applied them to a fresh copy of the repo. I'm not sure where all the extra commits came from, or how to make this branch clean. @timpau please check out #902.

yes, i guess something went wrong on that rebase. i can close this ?

**Pull request closed without merging**